extensions/wikia/AssetsManager/config.php

```diff
@@ -875,6 +875,8 @@
 	'type' => AssetsManager::TYPE_JS,
 	'assets' => [
 		'#group_oasis_jquery',
+		'//resources/wikia/libraries/mustache/mustache.js',
```

Comment:
this one is actually needed to make it work. I decided to include BannerNotifications instead of bypassing it as it's much simpler.